libs/.gitignore

```diff
@@ -6,6 +6,7 @@ node_modules/
 *.swn
 .DS_Store*
 test.js
+tests/assets/random.txt
```

Comment:
file was still committed - can you remove

Reply:
oops -- fixed that now